server/handlers/connections_handlers_test.go

```diff
@@ -0,0 +1,51 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	
+	"net/http"
+	"net/http/httptest"
+
+	"testing"
+
+	"github.com/layer5io/meshery/server/models"
+)
+
+var testcon = &models.ConnectionPayload{
+	Kind:    "kind-value",
+	SubType: "subtype-value",
+	Type:    "type-value",
+	MetaData: map[string]interface{}{
+		"key1": "value1",
+		"key2": 123,
+	},
+	Status: models.CONNECTED,
+	CredentialSecret: map[string]interface{}{
+		"secretKey1": "secretValue1",
+		"secretKey2": true,
+	},
```

Comment:
```suggestion
	CredentialSecret: map[string]interface{}{
		"token": "eyJhY2Nlc3NfdG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0ltdHBaQ0k2SW5CMVlteHBZenBvZVdSeVlTNXFkM1F1WVdOalpYTnpMWFJ2YTJWdUlpd2lkSGx3SWpvaVNsZFVJbjAuZXlKaGRXUWlPbHRkTENKamJHbGxiblJmYVdRaU9pSnRaWE5vWlhKNUxXTnNiM1ZrSWl3aVpYaHdJam94TmpnNU56TXlNRE16TENKbGVIUWlPbnQ5TENKcFlYUWlPakUyT0RrMk5EVTJNeklzSW1semN5STZJbWgwZEhBNkx5OXNiMk5oYkdodmMzUTZPVEF3TUM4aUxDSnFkR2tpT2lJek9UVmpOMlV3TnkwME1XTXhMVFF6T1RrdE9ERXhaQzFqWmpKak56ZGlabVZtTVRraUxDSnVZbVlpT2pFMk9EazJORFUyTXpJc0luTmpjQ0k2V3lKdmNHVnVhV1FpTENKdlptWnNhVzVsSWl3aWIyWm1iR2x1WlY5aFkyTmxjM01pWFN3aWMzVmlJam9pWWxkV2VtRkhNV2hqUlVKeldWaHNiR05xVlhWaFZ6ZzlPbUpYVm5waFIxWjVaVk14YW1KSE9URmFRVDA5SW4wLmpVelhmMkRrTi15NVpiWThSVkFuTjJZeXFTS05BT2tCZ0dDNHMxb2dnUFoxQS1CakdNbkFyMTZjRVBLcmVPVENXSzNfaU1DdWctcmltZG9QeGlqWFllX1d5bGpqeU9Ta1RyR0xpY0xubE1WNmhXNndJbXptT0E1VWh2UDVWZ09oeXRDZ1VQU3U0OFRVRFFTUE1NU1VkRnRFTzY2c0tvM3pXeURiV01lbm1rTHp6bGJwaFhFbXJpTDFhcnZkRHBFZV9ndHlWa0xjbktCS3p6U2Q2WWF3eDUzS0NhWGhGa2Q0Y0FVaUFVX2lGZjZnYW9ib0VRNUVfV2ZtRDJJaG9BaEtTWHJNbjdmTkxJTERwOFVXRUk3VGdwUUVFZ1QyY1VXdWY4eXlhV2NxOWpmRW9ia3ZQLXpTa29oXzJYZWlLRmtFRjAydTJ2VGp6Z3VpSmJRRl9DWmtQZGlVcGt1VlU0VEJHZVk5U2h6SWxUeGxYdWNySnp1UDI0YVJOMjl6cnNxN1hUMTktTVd2WVRkb3RCek1Dd3hhOTV2Q1BkdUFOTXhPTzVoYWJ3X2hfVUoyTGwycFFfT1dkeHJUSjFNV2pQR25hdzBFcGtqcVlDV1lSdzVJMnBwZGctWGRJUzM3aEpBbTk0MXpLNFhVanhIZzluTVJxSE53Y1RjWnFYRGU0UUstejBVWTJ0Nm0tNjBTZjZJRDg1anpkeFlxMmVfQ3BFMno3MFhaSmJZT1A1OGthU0Zfekxac1FUSkwyUFBDRmpGOGl2bnFFRzlzLUZqemRSSUxmUi1Tc2VsUjZzWk4tb3JteXlLbWF4MnlIYmljZkxTRjhveWQzWW5QSHk5ZWpYSkY1bVpwd3lPWUpmS1VRNEtOVTZ3QXg2cjljRnVvQ0NSRjJFRnJDdnJTVWE0IiwidG9rZW5fdHlwZSI6ImJlYXJlciIsInJlZnJlc2hfdG9rZW1iOiIteXI0Z2VDeEExWlJPTFJyeTlsa3doUXl5VGR4SzV4clFzNzY1ejBkYUxnLmEzSVdLVGZCVHQ5VzRkZ0Mwdl93U0dZX1ozYTZPSndWYl9jU1h4c1hUalUiLCJleHBpcnkiOiIyMDIzLTA3LTE5VDA3OjMwOjMyLjkxNjozNpswNToqMCA1",
	},
```

Reply:
Though this is not real token but just to mimic real data, I have added something which is closest to the real deal